blockmanager.go

```diff
@@ -1069,6 +1106,40 @@ func (b *blockManager) handleInvMsg(imsg *invMsg) {
 	}
 }
 
+// limitNumRejectedTxns limits the number of rejected transactions by evicting
+// a random rejected transaction if adding a new one would cause it to overflow
+// the max allowed.
+func (b *blockManager) limitNumRejectedTxns() error {
```

Comment:
I'd prefer this be defined above the only function it's called from so it's consistent with the vast majority of the code base.